python/main.py

```diff
@@ -62,6 +62,17 @@ def get_item():
 
     return {"item":items}
 
+@app.get("/search")
+def get_item(keyword):
+    logger.info(f"Search {keyword} items")
+    conn = sqlite3.connect(DBPATH)
+    conn.row_factory = dict_factory
+    c = conn.cursor()
+    c.execute("SELECT * FROM items WHERE name=?;", [keyword])
```

Comment:
Let's assume there's an item "long skirt". If you search "skirt", will the item "long skirt" be returned?

https://www.sqlitetutorial.net/sqlite-like/
The "LIKE" operator may be useful in this case.

Reply:
Thank you for pointing out this!!!
I did not notice the requirement is "including" the keyword.